src/test/java/edu/oakland/test/database/DatabaseGisInterfaceTest.java

```diff
@@ -0,0 +1,58 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.production.database.DatabaseGisInterfaceImplementation;
+import edu.oakland.production.database.DatabaseGisManagerStub;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database GisInterface Test")
+public class DatabaseGisInterfaceTest {
+
+  @Test
+  @DisplayName("LDP going in the same LDP coming out")
+  void dataPointInIsDataPointOut() {
+    DatabaseGisManagerStub dgms = new DatabaseGisManagerStub();
+    DatabaseGisInterfaceImplementation dgi = new DatabaseGisInterfaceImplementation(dgms);
+    LocationDataPoint ldp = new LocationDataPoint(0, 0, LocalDateTime.of(
+        (int) (Math.random() * 50 + 1970),
+        (int) (Math.random() * 12 + 1),
+        (int) (Math.random() * 28 + 1),
+        (int) (Math.random() * 24),
+        (int) (Math.random() * 60)
+    ));
+    int i = 0;
+    dgi.receiveStoreRequest(ldp);
+    assertEquals(null, dgms.getLocationDataPoint(i));
+  }
+
+  @Test
+  @DisplayName("Putting in the current satellite returns the new satellite")
+  void currentSatGetsNextSat() {
+    DatabaseGisManagerStub dgms = new DatabaseGisManagerStub();
+    DatabaseGisInterfaceImplementation dgi = new DatabaseGisInterfaceImplementation(dgms);
+    String currentSat = "";
+    assertEquals("", dgi.receiveNextSatRequest(currentSat));
+  }
+
+  @Test
+  @DisplayName("The Mode Requested is the Actual Mode")
+  void modeInIsModeOut() {
+    DatabaseGisManagerStub dgms = new DatabaseGisManagerStub();
+    DatabaseGisInterfaceImplementation dgi = new DatabaseGisInterfaceImplementation(dgms);
+    String n = "";
+    assertEquals(n, dgi.receiveModeRequest(n)); //supposed to return a String with the mode
+  }
+   
+  @Test
+  @DisplayName("Checks that manager isn't null")
+  void databaseGisManagerNotNull() {
+    DatabaseGisManagerStub dgms = new DatabaseGisManagerStub();
+    String currentSat = "";
+    assertEquals("", dgms.passNextSatRequest(currentSat));
```

Comment:
This doesn't actually check that the manager isn't null, as you're not actually testing the `DatabaseGisInterfaceImplementation` class. Can you model this test off of the similar test in `LocationDataPointTests` and `LocationDataPoint`?